### PR TITLE
Lift first commit

### PR DESCRIPTION
**Pull request by Soumyadeep-Oracle**

### CODE REVIEW

File: cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Package Name
+oradb_discovery_v3_6_11_19.sql
+
+## Description
+The Database discovery & analysis tool is a SQL utility that captures necessary details of an oracle database in order to analyse on-Prem or cloud Oracle Databases for the suitability of migration/upgrades. This utility runs on Non Container or Container databases and helps in generating HTML report with detailed description
+
+## Usage
+sqlplus "/ as sysdba"  @oradb_discovery_v3_6_11_19.sql
```
Comment (oheimburger) [inline cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/README.md:8]: Use ``` before and after code.

File: cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+### Step 1
+This is a discovery utility for Oracle database compatible for 11g onwards with a below features:
+1.	Current version of the script is v3.6 and it supports Oracle 11g or above database
+2.	Single version script for both Oracle Multitenant and  Non-Multitenant database
+3.	This is a platform independent script as we need sqlplus "/ as sysdba"  to execute only
+
+This script needs to execute on Oracle database server with a user who has “/as sysdba” access
```
Comment (oheimburger) [inline cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/README.md:16]: Use `/as sysdba`

File: cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+•	Copy the oradb_discovery_v3_6_11_19.sql file to the server
+•	To execute the script 	:  sqlplus "/ as sysdba"  @oradb_discovery_v3_6_11_19.sql
```
Comment (oheimburger) [inline cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/README.md:18]: Should read "To execute the script:"
Should read `sqlplus "/ as sysdba"  @oradb_discovery_v3_6_11_19.sql`

File: cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/files/LICENSE [hunk 4]
```diff
@@ -0,0 +1,35 @@
+Copyright (c) 2021 Oracle and/or its affiliates.
```
Comment (oheimburger) [inline cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/files/LICENSE:1]: This file should be moved one level up.

File: cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/files/oradb_discovery_v3_6_11_19.sql [hunk 5]
```diff
@@ -0,0 +1,2758 @@
...
+DBMS_OUTPUT.put_line('PROMPT Error!! This script is intended to execute on oracle database 11g or higher version.');
+delsqlfile;
+dbms_output.put_line('exit;');
+ELSE
+delsqlfile;
+END IF;
+END;
+/
+SPOOL OFF
+@version_not_matched.sql
```
Comment (oheimburger) [inline cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/files/oradb_discovery_v3_6_11_19.sql:70]: This file is not included. Where to find it?

File: cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/files/oradb_discovery_v3_6_11_19.sql [hunk 6]
```diff
@@ -0,0 +1,2758 @@
...
+   IF ( gv_slab in ('11g1','11g2','12c','13','18','19') ) THEN
+      dbms_output.put_line('Prompt Heading DB:DNFS Status Details');
+      dbms_output.put_line('select * from gv$dnfs_servers;');
+   END IF;
+   IF ( gv_slab in ('11g1','11g2','12c','13','18','19') ) THEN
+      dbms_output.put_line('Prompt Heading DB:Database ASM Disk Group Details');
+      dbms_output.put_line('SELECT o.name group_name, o.sector_size sector_size, o.block_size block_size, o.allocation_unit_size allocation_unit_size, o.state state, o.type type, o.total_mb total_mb, (total_mb - free_mb) used_mb, ROUND((1- (free_mb / decode(total_mb,0,1)))*100, 2)  pct_used ');
+	  dbms_output.put_line('FROM v$asm_diskgroup o ;');
+   END IF;
+   --IF ( gv_slab in ('11g1','11g2','12c','13','18','19') ) THEN
```
Comment (oheimburger) [inline cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/files/oradb_discovery_v3_6_11_19.sql:1685]: Can this code be removed?

File: cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/README.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
+# Package Name
+oradb_discovery_v3_6_11_19.sql
+
+## Description
+The Database discovery & analysis tool is a SQL utility that captures necessary details of an oracle database in order to analyse on-Prem or cloud Oracle Databases for the suitability of migration/upgrades. This utility runs on Non Container or Container databases and helps in generating HTML report with detailed description
```
Comment (oheimburger) [inline cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/README.md:5]: Change '&' to 'and'.
Change 'on-Prem' to 'on-premises'.
Always end a sentence properly, e.g., "in generating HTML report with detailed description."

File: cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/README.md (hunk 1 above)
Comment (Soumyadeep-Oracle) [reply cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/README.md:9]: Submitted with required changes.

File: cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/README.md (hunk 2 above)
Comment (Soumyadeep-Oracle) [reply cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/README.md]: Submitted with required changes.

File: cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/README.md (hunk 3 above)
Comment (Soumyadeep-Oracle) [reply cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/README.md]: Submitted with required changes.

File: cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/README.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+
+## License
+Copyright (c) 2023 Oracle and/or its affiliates.
+
+Licensed under the Universal Permissive License (UPL), Version 1.0.
+
+See [LICENSE](LICENSE) for more details.
+
+ORACLE AND ITS AFFILIATES DO NOT PROVIDE ANY WARRANTY WHATSOEVER, EXPRESS OR IMPLIED, FOR ANY SOFTWARE, MATERIAL OR CONTENT OF ANY KIND CONTAINED OR PRODUCED WITHIN THIS REPOSITORY, AND IN PARTICULAR SPECIFICALLY DISCLAIM ANY AND ALL IMPLIED WARRANTIES OF TITLE, NON-INFRINGEMENT, MERCHANTABILITY, AND FITNESS FOR A PARTICULAR PURPOSE.  FURTHERMORE, ORACLE AND ITS AFFILIATES DO NOT REPRESENT THAT ANY CUSTOMARY SECURITY REVIEW HAS BEEN PERFORMED WITH RESPECT TO ANY SOFTWARE, MATERIAL OR CONTENT CONTAINED OR PRODUCED WITHIN THIS REPOSITORY. IN ADDITION, AND WITHOUT LIMITING THE FOREGOING, THIRD PARTIES MAY HAVE POSTED SOFTWARE, MATERIAL OR CONTENT TO THIS REPOSITORY WITHOUT ANY REVIEW. USE AT YOUR OWN RISK.
```
[the remark this answers is not in the document]
Comment (Soumyadeep-Oracle) [reply cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/README.md]: Submitted with required changes.

File: cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/files/LICENSE (hunk 4 above)
Comment (Soumyadeep-Oracle) [reply cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/files/LICENSE]: Submitted with required changes.

File: cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/files/oradb_discovery_v3_6_11_19.sql (hunk 5 above)
Comment (Soumyadeep-Oracle) [reply cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/files/oradb_discovery_v3_6_11_19.sql:70]: Submitted with required changes.

File: cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/files/oradb_discovery_v3_6_11_19.sql (hunk 6 above)
Comment (Soumyadeep-Oracle) [reply cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/files/oradb_discovery_v3_6_11_19.sql]: Submitted with required changes.

File: cloud-architecture/custom-apps-and-consolidation/LICENSE
Comment (AlexanderHodicke) [inline cloud-architecture/custom-apps-and-consolidation/LICENSE:1]: License to 2023

**Pull request merged**